src/components/Main.js

```diff
@@ -1,19 +1,22 @@
 import "./Main.css";
 import ItemCard from "./ItemCard";
+import image01 from "../images/sunday.png";
+import image02 from "../images/red.png";
+import image03 from "../images/chocolate.png";
```

Comment:
You might want to do yourself a favour and name those images to something decriptive

Reply:
fair point ! changing these!